src/ast/ast_validations.c

```diff
@@ -916,26 +953,6 @@ static VISITOR_STRATEGY _Validate_shortest_path
 	return VISITOR_RECURSE;
 }
 
-// validate a pattern-path expression
-static VISITOR_STRATEGY _Validate_pattern_path
```

Comment:
why this was deleted?

Reply:
I believe this function only served the `CREATE` clause, which now explicitly calls `_Validate_CREATE_Entities`
by removing this we get the default behavior which simply recurse, so no real change.